CHANGELOG.md

```diff
@@ -70,7 +70,7 @@ Please see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest
 
 ## Next Release
 
-(no changes yet)
+- Bugfix: Ambassador now has appropriate permissions on IngressClass resources.
```

Comment:
Weirdly, this isn't actually necessary -- the YAML will get published live on merge.

Reply:
oh shoot, right. this was muscle memory mostly heh